### ISSUE TITLE
[from : ViennaUCT] bug for check_top_player

### ISSUE THREAD

**Issue by vineetsk1** [eternagame/EteRNA-public#306]
not sure how a bot detected this but :
 bug- http://screencast.com/t/9YRvCh0oGfuM

**Comment by vineetsk1** [comment]
@jeejee85 are the queries for bots different than the queries for humans? I am using "userpoints_get_current_points" to get the points of the player and then I am counting how many players have points that are above those points in the users database for the rank. Are bots still stored in the "users" database and their points are in "userpoints"? Or is ViennaUCT different?

**Comment by ElNando888** [comment]
The ViennaUCT account is a "normal" account. My software accesses the server the same way humans do. Only it does that with the cURL library instead of using a browser, that's all. And contrary to the other bots, it does try very hard to follow all the rules (constraints), even though it's using an engine that is a little different than the one in EteRNA (ViennaRNA v2.1.1 instead of 1.8.5) 
I sometimes log into this account to check the solutions it submitted, and for fun, I sometimes impersonate a sentient bot while I'm there. A few hours ago, I simply noticed something weird in the roadmap associated with Vinnie's account. It is my understanding that it could have happened with another "human" account.
Does this clarify?
